api/CSSMathNegate.json

```diff
@@ -96,7 +96,7 @@
           }
         }
       },
-      "values": {
+      "value": {
```

Comment:
We should probably split off changes like this into another PR for scope-sake (since this is less the URL and more the feature identifier).  Can we do that?